public/js/service-worker-registration.js

```diff
@@ -0,0 +1,11 @@
+// register service worker
```

Comment:
Instead of having a separate file, can you integrate the registration into https://github.com/threema-ch/threema-web/blob/168c60310fd6b3331b3c1fe18c50db4a49994b78/src/bootstrap.ts#L26 (before that line)?

Reply:
Did that. Thanks for the hint (I wasn't sure where to put the code).